packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/appeals/_appealNumber/controller.js

```diff
@@ -2,9 +2,9 @@ const { getAppealStatus } = require('#utils/appeal-status');
 const { formatCommentDeadlineText } = require('../../../../../utils/format-deadline-text');
 const { formatCommentDecidedData } = require('../../../../../utils/format-comment-decided-data');
 const { formatCommentHeadlineText } = require('../../../../../utils/format-headline-text');
-const { formatHeadlineData, formatRows } = require('@pins/common');
-const { appealSubmissionRows } = require('./ip-appeal-submission-rows');
-const { applicationRows } = require('./ip-application-rows');
+const { formatHeadlineData } = require('@pins/common');
+// const { appealSubmissionRows } = require('./ip-appeal-submission-rows');
+// const { applicationRows } = require('./ip-application-rows');
```

Comment:
Just wondering why these are commented out if the actual files/ functions have been deleted?

Reply:
mistake, will remove

Reply:
have removed these now